Backend/Remora.Discord.Rest/API/Applications/DiscordRestApplicationAPI.cs

```diff
@@ -57,6 +58,12 @@ IOptions<JsonSerializerOptions> jsonOptions
             _jsonOptions = jsonOptions.Value;
         }
 
+        /// <inheritdoc cref="DiscordHttpClient.WithCustomization"/>
```

Comment:
Don't think it'd be a appropriate to put all these on a base class?

Reply:
That's probably a good idea, yeah.